cl/lib/sqlcommenter.py

```diff
@@ -0,0 +1,74 @@
+from contextlib import ExitStack
+from typing import Any
+
+from django.db import connections
+
+
+def add_sql_comment(sql: str, **meta: Any) -> str:
+    """
+    Append a SQL comment containing key=value pairs from meta.
+    """
+    if not meta:
+        return sql
+
+    comment_parts = [
+        f"{key}={value}" for key, value in meta.items() if value is not None
```

Comment:
I performed the following test:

`/api/rest/v4/coverage/*scotus/`

I wanted to see what would happen if the URL already included the `/*` comment syntax.

I got this error:

```
unterminated /* comment at or near "/* user_id=8, url=/api/rest/v4/coverage/*scotus/ */"
LINE 1: ...t" WHERE "search_court"."id" = '*scotus' LIMIT 21 /* user_id...
```

Separately from this error, I’m also wondering whether this could be a security issue, specifically, if someone could use it to terminate the comment early and attempt SQL injection?

Can we escape `/*` and `*/` in the `meta.items` values (for example, by replacing them with `/ *` and `* /`)?